File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/ShowBoughtQuestItem.scala
package tools.packet.cashshop.operation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowBoughtQuestItem(private var _itemId: Int) extends PacketInput {
  def itemId: Int = _itemId

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/ShowBoughtQuestItem.java
package tools.packet.cashshop.operation;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowBoughtQuestItem(Integer itemId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}